dev-tools/cherrypick_pr

```diff
@@ -77,10 +79,10 @@ def main():
         check_call("git checkout -b {}".format(tmp_branch), shell=True)
         if call("git cherry-pick -x {}".format(" ".join(args.commit_hashes)),
                 shell=True) != 0:
-            print("Looks like you have cherry-pick errors.")
```

Comment:
Good catch! Do you prefer using `__future__` or ignoring the warning? https://docs.python.org/3.5/library/__future__.html